RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -1238,7 +1250,7 @@ private void parseAdobeMRUList(String regFileName, AbstractFile regFile, Buffere
                 line = line.trim();
             }
         }    
-        if (bbartifacts != null) {
+        if (bbartifacts.isEmpty()) {
```

Comment:
good catch on realizing it can't be null but could be empty fixing an existing bug.

However, I think we want a not empty here like you have below.

Reply:
Fixed